zebra/connected.c

```diff
@@ -284,6 +296,15 @@ void connected_up(struct interface *ifp, struct connected *ifc)
 		flags, &p, NULL, &nh, 0, zvrf->table_id, metric, 0, 0, 0,
 		false);
 
+	if (install_local) {
+		rib_add(afi, SAFI_UNICAST, zvrf->vrf->vrf_id, ZEBRA_ROUTE_LOCAL,
```

Comment:
Hmm. `metric` is really for the connected network… the `L` is local, does it make sense to unconditionally use `0` instead of `metric`?

Reply:
can you explain what you mean here?  I'm failing to see any meaningful distinction.  Not that I really care one way or the other as that this will only ever effect the metric displayed for the local route

Reply:
changed metric to 0